x-pack/plugins/security_solution/public/timelines/components/timeline/event_details.tsx

```diff
@@ -11,39 +11,67 @@
  */
 
 import { EuiSpacer } from '@elastic/eui';
-import React from 'react';
+import React, { useMemo } from 'react';
 import deepEqual from 'fast-deep-equal';
+import { findIndex } from 'lodash/fp';
 
 import { BrowserFields, DocValueFields } from '../../../common/containers/source';
 import {
   ExpandableEvent,
   ExpandableEventTitle,
+  HandleOnEventClosed,
 } from '../../../timelines/components/timeline/expandable_event';
 import { useDeepEqualSelector } from '../../../common/hooks/use_selector';
+import { useTimelineEventsDetails } from '../../containers/details';
+import { timelineSelectors } from '../../store/timeline';
+import { timelineDefaults } from '../../store/timeline/defaults';
 
 interface EventDetailsProps {
   browserFields: BrowserFields;
   docValueFields: DocValueFields[];
   timelineId: string;
+  handleOnEventClosed?: HandleOnEventClosed;
 }
 
 const EventDetailsComponent: React.FC<EventDetailsProps> = ({
   browserFields,
   docValueFields,
   timelineId,
+  handleOnEventClosed,
```

Comment:
make sure to add the prop here as well https://github.com/elastic/kibana/pull/83963/files#diff-11db78c0109cf0169d6f9259ac4b04999072a6b977771aee2e0e8344ddafeeadR86